.circleci/config.yml

```diff
@@ -53,4 +50,4 @@ workflows:
     jobs:
       - lint-python
       - test-python
-      - test-site
+#     - test-site
```

Comment:
Why commented out? 'Cos it's currently broken?

Reply:
Because `/` is returning a 401, I will try to use `/_status/check`.

Reply:
Testing that `/` returns a 401 would be another option, either seems fine.